samples/ControlCatalog.NetCore/Program.cs

```diff
@@ -118,6 +118,13 @@ public static AppBuilder BuildAvaloniaApp()
                 })
                 .UseSkia()
                 .UseManagedSystemDialogs()
+                .AfterSetup(_ =>
```

Comment:
Maybe this would be better? (not sure if it works, but reduces globals usage)

```cs
.AfterSetup(builder =>
{
    builder.Instance.AttachDevTools(...);
})
```

Reply:
Yes, it's better. Thanks!